powerauth-nextstep-client/src/main/java/io/getlime/security/powerauth/lib/nextstep/client/NextStepClient.java

```diff
@@ -2356,6 +2356,7 @@ public ObjectResponse<GetUserAuthenticationListResponse> getUserAuthenticationLi
      * @throws NextStepClientException Thrown when REST API call fails, including {@link ErrorResponse} with error code.
      * @see #createCredential(CreateCredentialRequest)
      */
+    @Deprecated
```

Comment:
When using `@Deprecated`, also document it in JavaDoc as `@deprecate` what to use instead.